source/linux/networking/connecting.rst

```diff
@@ -0,0 +1,58 @@
+Connection to other Computers
+=============================
+
+ssh
+^^^
+    Description:
+        Starts a **s** ecure **s** hell sessions with the given host and user.
+        When the connection is established, you are working on the remote system.
+
+    Usage:
+        .. code-block:: bash
+
+            host$ ssh [user@]host
+
+    Example:
+        .. code-block:: bash
+
+            host$ ssh user@10.0.0.137
```

Comment:
We should use our standard IP address to connect to our target here: 192.168.3.11 
And not any random address.